SETUP.md

```diff
@@ -49,6 +49,27 @@ On Mac OSX and Linux, use `./gradlew` instead of `gradlew`.
 
 If you have errors with a package missing please make sure you have setup your environment, and are using Oracle JDK 8.
 
+To check which java you are using do 
+`java -version` in a command prompt or terminal.
+If you are using Oracle JDK 8+ it will not work because the Java distributions above JDK 8 using may not have the needed javax classes.
+
+Open JDK 8 download: https://openjdk.java.net/install/
+#### macOS guide
+In order to get JDK 8, run the following command:
+`% /usr/libexec/java_home -V`
+If you see something like
+
+`% 1.8.0_202, x86_64:	"Java SE 8"	/Library/Java/JavaVirtualMachines/jdk1.8.0_202.jdk/Contents/Home`
```

Comment:
This version shouldn't be hardcoded, just indicate they have to change the version. You'd be surprised how many people just blindly copy and paste commands

Reply:
true. Changing path can be a little tricky though and I wanted to make it easier to find. But which line do you suggest i remove/edit?

Reply:
```diff
- `% 1.8.0_202, x86_64:	"Java SE 8"	/Library/Java/JavaVirtualMachines/jdk1.8.0_202.jdk/Contents/Home`
+ `% 1.8.0_VERSION, x86_64:	"Java SE 8"	/Library/Java/JavaVirtualMachines/jdk1.8.0_VERSION.jdk/Contents/Home`
```

Reply:
changed it